model/jpa/src/main/resources/META-INF/jpa-changelog-2.4.1.xml

```diff
@@ -22,4 +22,46 @@
          <customChange class="org.keycloak.connections.jpa.updater.liquibase.custom.MigrateUserFedToComponent"/>
      </changeSet>
 
+    <changeSet author="glavoie@gmail.com" id="2.4.1.index">
+        <preConditions onSqlOutput="TEST" onFail="MARK_RAN">
```

Comment:
What's with the pre-coniditions? Would be better to just add same indexes for all dbs